logical/framework/backend.go

```diff
@@ -280,6 +286,16 @@ func (b *Backend) Type() logical.BackendType {
 	return b.BackendType
 }
 
+// RegisterLicense performs backend license registration.
+func (b *Backend) RegisterLicense(license interface{}) error {
+	if b.LicenseRegistration != nil {
+		if err := b.LicenseRegistration(license); err != nil {
```

Comment:
Probably there is other code that is also similarly verbose, but there's a simpler way to write this function:

```go
if b.LicenseRegistration == nil {
    return nil
}
return b.LicenseRegistration(license)
```